index.html

```diff
@@ -4,11 +4,11 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Document </title>
     <link rel="stylesheet" href="style.css"> 
 </head>
 <body>
-    <h1>Hello  Microverse </h1>
+    <h1>Hello Microverse </h1>
 
     
```

Comment:
## 6.
### Your MIT.md file link
![MIT-LINK](https://user-images.githubusercontent.com/93039359/184927017-4d7f7a32-6bff-45b5-afc7-30e784732957.png)

- [ ] Your readme file doesn't have a good link of your MIT.md file. Please fix it something like below :arrow_down:
`This project is [MIT](./MIT.md) licensed.`